securedrop_client/api_jobs/uploads.py

```diff
@@ -93,20 +93,21 @@ def call_api(self, api_client: API, session: Session) -> str:
             # Update following draft replies for the same source to reflect the new reply count
             draft_file_counter = draft_reply_db_object.file_counter
             draft_timestamp = draft_reply_db_object.timestamp
+
             update_draft_replies(
                 session,
                 source.id,
                 draft_timestamp,
                 draft_file_counter,
                 new_file_counter,
-                commit=False,
```

Comment:
Why change this? Is there a reason to commit before adding the reply to replies table, bumping the source interaction count, and deleting the draft reply?

Reply:
You're right; I changed when I was concerned that the delayed commits were contributing to the interaction_count being wrong and I was trying to narrow it down as much as possible, but that was before I just decided to sort all the pending replies to the bottom regardless of their interaction_count. I'll remove this change.

Reply:
This change was reverted in cda3150